test/e2e/apps/cronjob.go

```diff
@@ -403,7 +409,9 @@ var _ = SIGDescribe("CronJob", func() {
 			[]byte(`{"metadata":{"annotations":{"patchedstatus":"true"}},"status":`+string(cjStatusJSON)+`}`),
 			metav1.PatchOptions{}, "status")
 		framework.ExpectNoError(err)
-		framework.ExpectEqual(patchedStatus.Status.LastScheduleTime.Equal(&now1), true, "patched object should have the applied lastScheduleTime status")
+		if !patchedStatus.Status.LastScheduleTime.Equal(&now1) {
```

Comment:
Note that I am assuming that the intent here was to do the equal comparison with `patchedStatus.Status.LastScheduleTime.Equal`, either to exercise that function or because it does something special that `framework.ExpectEqual` wouldn't do.

Either way, not changing the logic of the test makes sense, so :+1: